sql/core/src/test/resources/sql-tests/inputs/pgSQL/boolean.sql

```diff
@@ -17,19 +17,19 @@ SELECT 1 AS one;
 
 SELECT true AS true;
 
-SELECT false AS false;
+SELECT false AS `false`;
```

Comment:
Rewrite `AS false` to ```AS `false` ``` to reduce changes.

Reply:
Shall we add `SPARK-28349` as a comment before this?

Reply:
Since there is many instances, let's add that comment at the top of this file.